### PR TITLE
docs: Update `getting-started.md` 

### PR DESCRIPTION
**Pull request by blueberryapple**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ x ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Updated getting started command to include instructions for yarn 2+.

### CODE REVIEW

File: docs/src/user-guide/getting-started.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ You can install and configure ESLint using this command:
 
 ```shell
 npm init @eslint/config
+
+# or
+
+yarn dlx @eslint/create-config
```
Comment (harish-sethuraman) [inline docs/src/user-guide/getting-started.md:30]: This was intentional we didn't want to add all package managers for initializing the config. https://github.com/eslint/eslint/pull/16084#discussion_r915359596

Comment (blueberryapple) [reply docs/src/user-guide/getting-started.md:30]: Then we should probably remove the yarn command here too:
https://github.com/eslint/eslint/blob/3cbe11f260c0206cb422778fef4aa85433b389eb/docs/src/user-guide/getting-started.md?plain=1#L35-L43

Comment (harish-sethuraman) [reply docs/src/user-guide/getting-started.md:30]: Yea good catch, may be we can remove it. But would like to hear from others as well :)

Comment (nzakas) [reply docs/src/user-guide/getting-started.md:30]: The installation case is different than the execution case. For installation, the command will ask if you want to install using npm, yarn, pnpm, etc. so it doesn’t really matter how you run it. I think for executing eslint it make sense to also mention yarn.

**Comment by nzakas** [comment]
We purposely left this out, so closing.

**Pull request closed without merging**